### ISSUE TITLE
Pexels Free Photo Quality Degraded as Featured Image

### ISSUE THREAD

**Issue by sdixon194** [Automattic/wp-calypso#46420]
When adding photos from Pexels as a featured image on posts, the image quality is noticeably degraded on the live site. Reproducible on both simple and atomic sites, although turning off Photon on atomic sites fixes the issue. Downloading the image from the Pexels website directly and uploading causes the same issue. Uploading other images doesn't seem to cause an issue.

#### Steps to reproduce

1. Create a New Post
2. Add a featured image
3. Select a photo from the Pexels free photo library
4. See the quality of the image degrade on the live site.

#### What I expected

The image quality to be retained, like this:

![good quality image](https://user-images.githubusercontent.com/33553323/95901273-13cb2000-0d61-11eb-86a7-50b38fbbb411.png)

#### What happened instead

The image quality is clearly degraded and you can see blurriness and aliasing:

![aliasing image](https://user-images.githubusercontent.com/33553323/95901223-031aaa00-0d61-11eb-9bcf-745a8ca1dadd.png)

**Comment by noahtallen** [comment]
> Downloading the image from the Pexels website directly and uploading causes the same issue

Perhaps this just means that Pexels doesn't offer high quality download? Does the downloaded image have the same compression issues before upload?

**Comment by sdixon194** [comment]
Nope, seems to be fine. And even when [viewing the image URL, it seems fine](https://middlemanlab.files.wordpress.com/2020/10/pexels-photo-1001990-1.jpeg). 

But once it's added as the featured image, things get weird. The thumbnails in the media library also look quite degraded:

![Screen Shot on 2020-10-23 at 09:17:00](https://user-images.githubusercontent.com/33553323/97008228-8f229380-1510-11eb-9281-cea8b41f8359.png)

I wonder if maybe it's related to Photon Subsizes?

**Comment by ericamQA** [comment]
### Removing, stale label. 

I tested and was able to reproduce on some themes (Dara) but not others (Veni, Olsen) 

**Dara (quality degraded)**

![Dara-pexels-quality](https://user-images.githubusercontent.com/44411833/115911398-098a8c80-a43c-11eb-8c95-ababdf7d53d0.png)

**Veni (quality retained)**

![Veni-Pexels-Quality](https://user-images.githubusercontent.com/44411833/115911510-2de66900-a43c-11eb-8efb-c625b461faf9.png)

**Olsen (quality retained)**

![Olsen-Pexels-Quality](https://user-images.githubusercontent.com/44411833/115911588-46568380-a43c-11eb-9b6f-6b2fbdad21f3.png)

**Comment by katiebethbrown** [comment]
Had a similar discussion in Slack ( p1619358835492800-slack-C3GP81E05 ), regarding a site where the cover and image blocks show a noticeable decrease in quality over the originals. Some digging turned up this related issue:

https://github.com/Automattic/jetpack/issues/10984

What appears to be happening is that Photon is calculating the source image size based on the Media Width set in the theme. For Varia/child themes that's 750px, even though with full-width blocks elements on the live site can be much wider than that.

Setting a custom, wider width in the Additional CSS tab does appear to increase the quality of the image served from photon without turning it off.

<img width="282" alt="mediawidth" src="https://user-images.githubusercontent.com/42279181/116007308-61f09400-a5cc-11eb-89df-22765bb8f1d2.png">
